core/object.c

```diff
@@ -302,14 +302,14 @@ lai_api_error_t lai_obj_to_string(lai_variable_t *out, lai_variable_t *object, s
         size_t buffer_length = 0;
         uint8_t *buffer = lai_exec_buffer_access(object);
         for(uint64_t i = 0; i < lai_exec_buffer_size(object); i++){
-            if(buffer[i] == '\0') 
+            if(buffer[i] == '\0')
                 break;
             buffer_length++;
         }
-        
+
         if(buffer_length == 0){
             lai_create_string(out, 0);
-        } else if(size == ~(uint64_t)(0)){
+        } else if(size == ~(uint32_t)(0)){
```

Comment:
Since `size` is a size_t here, wouldn't it make more sense casting to a size_t so it works correctly in both 64 and 32bit mode?